admin/view/template/module/tawkto.twig

```diff
@@ -195,6 +195,34 @@
 							</div>
 						</div>
 						<br></br>
+						<div class="col-lg-12">
+							<div class="panel-heading"><strong>Security Settings</strong></div>
+						</div>
+						<div class="form-group col-lg-12 row">
+							<label for="secure_mode_enabled" class="col-xs-6 control-label">
+								Secure Mode Enabled
+								<br>
+								<span class="attrib_desc">If Secure Mode is enabled on your property, please enter your
+									Javascript API Key to ensure visitor recognition works correctly.</span>
+							</label>
+							<div class="col-xs-6 control-label ">
+								<input type="checkbox" class="control-label col-xs-6" name="secure_mode_enabled"
+									id="secure_mode_enabled" value="1"
+									{{(hierarchy[0].security_opts.secure_mode_enabled)?'checked':''}} />
+							</div>
+						</div>
+						<div class="form-group col-lg-12 row">
+							<label for="js_api_key" class="col-xs-6 control-label">
+								Javascript API Key
+								<br>
+								<span class="attrib_desc">Ensure hash is generated for secure mode</span>
```

Comment:
"Used to authenticate client side API calls when secure mode is enabled for property"